frontend/src/App.scss

```diff
@@ -53,3 +53,20 @@ div.bootstrap-table {
 .table .tr .td {
   word-break: break-word;
 }
+
+//by default highlight required fields in blue.
+.form-group.required input {
```

Comment:
could maybe use :first-child here instead?

Reply:
something like this:
.form-group.required > *:first-child

Reply:
Just tried this, didn't work because a lot of the groups have the label as the first child, so the label gets the border rather than the input field